nussl/datasets/transforms.py

```diff
@@ -3,6 +3,7 @@
 import logging
 import random
 from collections import OrderedDict
+import librosa
```

Comment:
Why is there a new import here? I don't see where librosa is called later on. I think this is a stray import that you don't need anymore.